.github/workflows/app-build-verify.yml

```diff
@@ -254,29 +264,39 @@ jobs:
     # alive.
     #
     # Only runs when target platform and format are explicitly Linux and AppImage.
-    - name: Build AppImage project
+    - name: Build AppImage Project
       if: >
         startsWith(inputs.runner-os, 'ubuntu')
         && contains(fromJSON('["Linux"]'), inputs.target-platform)
         && contains(fromJSON('["AppImage"]'), inputs.target-format)
+      working-directory: ${{ steps.create.outputs.project-path }}
       run: |
-        cd tests/apps/verify-${{ inputs.framework }}
-        briefcase create linux AppImage
+        # PyGObject>=3.46.0 requires a version of glibc that isn't available in manylinux images;
+        # so, the version will need to be constrained to successfully build an AppImage with Toga.
+        # Furthermore, Toga>0.3.1 requires PyGObject>=3.46.0 so its version is constrained as well.
+        if [ "${{ startsWith(inputs.framework, 'toga') }}" = "true" ]; then
+          CONFIG_OVERRIDE_REQUIRES='--config requires=["toga-gtk==0.3.1","PyGobject<3.46.0"]'
+        fi
+
+        briefcase create linux AppImage \
+          ${{ steps.output-format.outputs.template-override }} \
+          ${CONFIG_OVERRIDE_REQUIRES}
         briefcase build linux AppImage
         briefcase package linux AppImage --adhoc-sign
 
-    - name: Build Flatpak project
+    - name: Build Flatpak Project
       if: >
         startsWith(inputs.runner-os, 'ubuntu')
         && contains(fromJSON('["", "Linux"]'), inputs.target-platform)
         && contains(fromJSON('["", "Flatpak"]'), inputs.target-format)
         && contains(fromJSON('["Toga", "PyGame"]'), inputs.framework)
+      working-directory: ${{ steps.create.outputs.project-path }}
       run: |
-        sudo apt-get update -y
-        sudo apt-get install -y flatpak flatpak-builder
+        sudo apt update -y
+        sudo apt install -y --no-install-recommends flatpak flatpak-builder elfutils
```

Comment:
What is elfutils for? Why the new requirement?

Reply:
I added `--no-install-recommends` to avoid all the stuff we don't need that `apt` installs by default.

As for `elfutils`, it's only a recommended package for `flatpak-builder` but it's required because it contains `eu-strip` which `flatpak-builder` needs.